libraries/AC_Avoidance/AP_OAPathPlanner.cpp

```diff
@@ -35,9 +35,9 @@ const AP_Param::GroupInfo AP_OAPathPlanner::var_info[] = {
     // @Param: TYPE
     // @DisplayName: Object Avoidance Path Planning algorithm to use
     // @Description: Enabled/disable path planning around obstacles
-    // @Values: 0:Disabled,1:BendyRuler,2:Dijkstra,3:Dijkstra with BendyRuler
+    // @Values: 0:Report Only,1:BendyRuler,2:Dijkstra,3:Dijkstra with BendyRuler
     // @User: Standard
-    AP_GROUPINFO_FLAGS("TYPE", 1,  AP_OAPathPlanner, _type, OA_PATHPLAN_DISABLED, AP_PARAM_FLAG_ENABLE),
+    AP_GROUPINFO("TYPE", 1,  AP_OAPathPlanner, _type, OA_PATHPLAN_REPORT_ONLY),
```

Comment:
You've removed the enable flag, we don't want to do that. We also probably don't want to change the current default and surprise users. The only thing required here is a change in the parameter "values" description.